tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/chunk/ChunkReader.java

```diff
@@ -152,11 +153,13 @@ private PageReader constructPageReaderForNextPage(int compressedPageBodyLength)
               + Arrays.toString(compressedPageBody) + ". Actual:" + chunkDataBuffer
               .remaining());
     }
-
     chunkDataBuffer.get(compressedPageBody, 0, compressedPageBodyLength);
     valueDecoder.reset();
-    PageReader reader = new PageReader(ByteBuffer.wrap(unCompressor.uncompress(compressedPageBody)),
-        chunkHeader.getDataType(),
+    ByteBuffer pageData = ByteBuffer.wrap(unCompressor.uncompress(compressedPageBody));
+    if (endianType.equals("LITTLE_ENDIAN")) {
```

Comment:
It would be better to make "LITTLE_ENDIAN" a constant or an enum.

Reply:
> It would be better to make "LITTLE_ENDIAN" a constant or an enum.

I have changed "LITTLE_ENDIAN" to an enum.